neon-animation-behavior.html

```diff
@@ -37,9 +37,10 @@
 
     },
 
-    registered: function() {
-      new Polymer.IronMeta({type: 'animation', key: this.is, value: this.constructor});
```

Comment:
This is strictly true. However, since the the use of `iron-meta` here was not documented and the change is performance positive, I'm inclined to do some usage testing before deciding to retain this. Also, to make the element compatible with `lazyRegister` (something that seems worth doing), the value stored would have to be the element name rather than the constructor (which is unavailable at initial registration).